membership-attribute-service/app/models/SupportReminders.scala

```diff
@@ -25,6 +25,7 @@ object RecurringReminderStatus {
 
 case class SupportReminderDb (
   isCancelled: Boolean,
+  reminder_code: String,
```

Comment:
maybe I should camel case `isCancelled` for consistency?

Reply:
I'm just gonna go ahead and do that, I can't face the mixed casing 😂

Reply:
👍